bin/nodetool

```diff
@@ -8,6 +8,8 @@
 %% -------------------------------------------------------------------
 -mode(compile).
 
+-define(SHUTDOWN_TIMEOUT_MS, 300_000).
```

Comment:
**5 minutes** is too long for user to wait.
Mostly users will `ctrl + c ` to stop waiting for so long time with panic.
Maybe we should spawn a child process to print the stop process is ongoing.

Reply:
Thanks @zhongwencool, added a periodic status messages while EMQX is stopping: 
```
# bin/emqx stop
EMQX is shutting down, please wait...
EMQX is shutting down, please wait...
EMQX is shutting down, please wait...
EMQX is shutting down, please wait...
ok
```

Reply:
After some more testing, I decreased the timeout from 5 to 2 min. I wasn't able to re-produce the case when 2min was not enough to shutdown gracefully. 
2 min also coincides with the default wait period in EMQX script:
https://github.com/emqx/emqx/blob/master/bin/emqx#L1020

esockd_connection_sup spends the most time terminating its children, we can try to make it even more 'brutal' (kill all connections and don't wait for down messages) but I'm reluctant to change it solely for the purpose of speeding up graceful shutdown. It would look less graceful if we go for it...